controllers/dspipeline_params.go

```diff
@@ -99,15 +102,19 @@ func (p *DSPAParams) UsingTektonEngineDriver(dsp *dspa.DataSciencePipelinesAppli
 // explicitly set images
 func (p *DSPAParams) GetImageForComponent(dsp *dspa.DataSciencePipelinesApplication, v1Image, v2ArgoImage, v2TektonImage string) string {
 	if p.UsingV2Pipelines(dsp) {
-		if p.UsingArgoEngineDriver(dsp) {
-			return v2ArgoImage
-		} else {
-			return v2TektonImage
-		}
+		return p.GetImageForComponentV2(dsp, v2ArgoImage, v2TektonImage)
 	}
 	return v1Image
 }
 
+func (p *DSPAParams) GetImageForComponentV2(dsp *dspa.DataSciencePipelinesApplication, v2ArgoImage, v2TektonImage string) string {
```

Comment:
Rather than add a new function that 'competes' with the overall GetImage method that already exists, I feel like we should instead just pass "" in to the GetImageForComponent v1 parameter

Reply:
Also FWIW, this could be handled here in a later ticket: https://github.com/opendatahub-io/data-science-pipelines-operator/issues/502